src/core/js/libraries/jquery.a11y.js

```diff
@@ -221,36 +220,49 @@
 
                 //SEARCH FOR TEXT ONLY NODES AND MAKE TABBABLE
                 var newChildren = [];
-                var added = false;
+                var currentText = "";
                 for (var i = 0; i < $element[0].childNodes.length; i++) {
                     var child = $element[0].childNodes[i];
                     var cloneChild = $(child.outerHTML)[0];
-                    switch(child.nodeType) {
-                    case 3: //TEXT NODE
-                        // preserve whitespace in ie8 by adding initial zero-width space
-                        var childContent = child.textContent || "&#8203;" + child.nodeValue;
-                        //IF TEXT NODE WRAP IN A TABBABLE SPAn
-                        newChildren.push( makeElementTabbable($("<span>"+childContent+"</span>")) );
-                        added = true;
-                        break;
-                    case 1: //DOM NODE
-                        var $child = $(cloneChild);
-                        if (($child.is(domSelectors.wrapStyleElements) && !added) || $child.is(domSelectors.wrapIgnoreElements)) {
-                            //IGNORE NATIVELY TABBABLE ELEMENTS AND STYLING ELEMENTS
-                            newChildren.push( $child );
-                        } else {
-                            var childChildren = $child.children();
-                            if (childChildren.length === 0) {
-                                //DO NOT DESCEND INTO TEXT ONLY NODES
-                                var textContent = $child.text();
-                                if (stringTrim(textContent) !== "") makeElementTabbable($child);
+                    switch (child.nodeType) {
+                        case 3: //TEXT NODE
+                            //IF TEXT NODE APPEND TO CURRENT TEXT VALUE
+                            currentText += child.textContent;
+                            break;
+                        case 1: //DOM NODE
+                            var $child = $(cloneChild);
+                            //IGNORE STYLING ELEMENTS
+                            if ($child.is(domSelectors.wrapStyleElements)) {
+                                currentText += child.outerHTML;
                             } else {
-                                //DESCEND INTO NODES WITH CHILDREN
-                                makeChildNodesAccessible($child);
+                                //APPEND CURRENT TEXT VALUE BEFORE CONTINUING
+                                if (currentText.length > 0) {
+                                    //MAKE TABBABLE IF THE TEXT CONTAINS CHARACTERS OTHER THAN FULL STOPS, COMMAS & SPACES
```

Comment:
this might be a bit too English language specific? why did you choose these characters and are there any potential others in different languages that could have the same impact on a screen reader?